gym/tf_od_api/workspace/train_core/eval.sh

```diff
@@ -0,0 +1,9 @@
+cd /opt/tf_od_api/workspace/train_core
+
+echo "Evaluating the model"
+
+python model_main_tf2.py \
```

Comment:
Just like in the `train_core/core.sh`, I think we can define a variable at the beginning of the script with the exact folder we are using to store the models

Reply:
this has been solved in the recent commit